src/concrete/ml/quantization/quantized_module.py

```diff
@@ -252,7 +252,9 @@ def post_processing_params(self, post_processing_params: Dict[str, Any]):
         self._post_processing_params = post_processing_params
 
     # pylint: disable-next=no-self-use
-    def post_processing(self, values: numpy.ndarray) -> numpy.ndarray:
+    def post_processing(
```

Comment:
great, this is going to combine well with what I've done for quantized modules in https://github.com/zama-ai/concrete-ml/pull/660/files#diff-84c5dca0cde5235ebd72ca3a3d3ee9e9ca1c4d17e9e28ed7b92885705e5ae346